### PR TITLE
Introduce ProjectDashboard md object containing dashboard tab basic info

### PR DESCRIPTION
**Pull request by peter-plochan**
resolves issue #283

### CODE REVIEW

File: src/main/java/com/gooddata/md/ProjectDashboard.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public static class Tab {
+
+        private final String identifier;
+        private final String title;
+
+        @JsonCreator
+        public Tab(@JsonProperty("identifier") String identifier, @JsonProperty("title") String title) {
```
Comment (martiner) [inline src/main/java/com/gooddata/md/ProjectDashboard.java]: does it have to be public?

File: src/main/java/com/gooddata/md/ProjectDashboard.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @JsonIgnore
+    public Tab getTabByName(String name) {
+        for (Tab tab : getTabs()) {
+            if (Objects.equals(tab.title, name)) {
+                return tab;
+            }
+        }
+        return null;
```
Comment (martiner) [inline src/main/java/com/gooddata/md/ProjectDashboard.java]: document that method may return null

Comment (peter-plochan) [inline src/main/java/com/gooddata/md/ProjectDashboard.java]: documented

**Comment by martiner** [comment]
Hm, apparently "fix for" doesn't close corresponding issue, list of keywords is https://help.github.com/articles/closing-issues-via-commit-messages/

**Pull request merged**